src/gwt/src/org/rstudio/studio/client/common/spelling/TypoSpellChecker.java

```diff
@@ -398,6 +409,26 @@ public boolean shouldCheckSpelling(DocDisplay dd, Range r)
       return true;
    }
 
+   /*
+      Stop gap function to prevent loading dictionaries that Typo.js has
+      severe issues with. This is being tracked to be fixed in issue #6041 as
+      soon as possible so this can then be removed.
+    */
+   private static String[] realtimeDictBlacklist = {"cs_CZ", "de_DE_neu", "lt_LT", "pt_BR"};
```

Comment:
I spot checked a few other dictionaries and they seemed ok, these are the ones listed in the bug. Adding more to this list will be easy and the intention is to not have this problem for too long. If we decide to not spend more time working on this issue, which could be reasonable considering all the variables, I'd move this to a more permanent external home.

By removing `canRealtimeSpellcheckDict` we can easily be pointed to and clear out all of the subsequent cruft that uses it.